SocketRocket/SRWebSocket.m

```diff
@@ -1570,7 +1567,7 @@ - (NSOperationQueue *_Nullable)delegateOperationQueue
 
 @end
 
-//#define SR_ENABLE_LOG
+#define SR_ENABLE_LOG
```

Comment:
Should be disabled on the current master branch.
Let me know if it's not. Shouldn't be enabled by default.